schemas/string_interpolation/transforms/format_date_range_pair.json

```diff
@@ -0,0 +1,36 @@
+{
+  "$id": "https://eq.ons.gov.uk/string_interpolation/transforms/format_date_range_pair.json",
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "format_date_range": {
+    "description": "A transform for formatting a pair of dates representing the start and end of a date range.",
+    "type": "object",
+    "properties": {
+      "transform": {
+        "type": "string",
+        "enum": ["format_date_range_pair"]
```

Comment:
The `_pair` could be dropped, I don't think it is necessary.

Reply:
There is already the old version of this which is `format_date_range` - should it be superseded or does this need a new name?

Reply:
So it doesn't look like we have a transform `format_date_range` in runner, just validator 🤔 . Given the function that passes input into this has to be a list of inputs, I guess this should supersede the existing `format_date_range`.